cypress/README.md

```diff
@@ -52,16 +47,6 @@ We have implemented a new approach for cypress tests, using cypress-react framew
 4. To run specific Cypress tests at the command line (headless browser for continuous integration) run `npx cypress run --component --spec './src/components/[component]/[fileName].test.js'`. Test results can be seen in the console run summary.
```

Comment:
Might be worth updating this command to account for typescript test files as well?

```suggestion
4. To run specific Cypress tests at the command line (headless browser for continuous integration) run `npx cypress run --component --spec "cypress/components/[component]/*.test.*`. Test results can be seen in the console run summary.
```